docs/deployment/index.mdx

```diff
@@ -1,5 +1,93 @@
-# Deployment
+# Deployment Guide
+
+Topaz is most easily deployed as a container, either as a sidecar or as a microservice in the same region / AZ as the application it supports.
+
+For environments that don't support running container images, Topaz can be run as a binary.
 
 ## Deploy as a Sidecar
 
+WIP
```

Comment:
@apoland I was hoping you could add some basic instructions on how to configure Topaz as a sidecar? I don't think we have any existing docs on this.

Reply:
I'm happy to merge this one and then have you open a new PR for that